pytroll_collectors/segments.py

```diff
@@ -489,6 +524,23 @@ def _find_time_slot(self, time_obj):
 
         return str(time_obj)
 
+    def check_schedule_time(self, checkTime, rawHour, rawMinute):
+        # Check if raw time is inside configured interval
+        toret = 0
+
+        # Convert check time into int variables
+        rawT = (60 * rawHour) + rawMinute
+        if checkTime["midnight"] and rawT < checkTime["start"]:
+            rawT += 24*60
+
+        # Check start end time
+        if rawT >= checkTime["start"] and rawT <= checkTime["end"]:
+            # Raw time in range, check interval
+            if ((rawT - checkTime["start"]) % checkTime["delta"]) == 0:
+                toret = 1
+
+        return toret
```

Comment:
Python uses boolean types, like True and False. Please use that instead of 0 and 1.